controller_manager/doc/controller_chaining.rst

```diff
@@ -48,8 +48,12 @@ This means the following:
 
 .. note::
 
-  Only the controllers that exposes the reference interfaces are switched to chained mode, when their reference interfaces are used by other controllers. When their reference interfaces are not used by the other controllers, they are switched back to get references from the subscriber.
-  However, the controllers that exposes the state interfaces are not triggered to chained mode, when their state interfaces are used by other controllers.
+  Controllers that expose the reference interfaces are switched to chained mode only when their reference interfaces are used by other controllers. When their reference interfaces are not used by other controllers, they are switched back to get references from the subscriber.
+  However, the controllers that expose the state interfaces are not triggered to chained mode when their state interfaces are used by other controllers.
+
+.. note::
```

Comment:
I just rendered it, '<text>' looks ok (and is used throughout the page already). Should we move the note further up the page, we use preceding controller already above?

Reply:
I thought about this. My view is that the reader doesn't need to exactly know what *preceding* means in the first few paragraphs. Having a gist is enough. But for the rest of the document, a proper definition is needed.